origin-js/src/resources/marketplace.js

```diff
@@ -176,17 +171,30 @@ export default class Marketplace {
    * @param {string} listingId
    * @param {{blockNumber: integer, logIndex: integer}} blockInfo - Optional argument
    *   to indicate a specific version of the listing should be loaded.
+   * @param opts: {loadOffers: boolean} also load offers for this listing
    * @returns {Promise<Listing>}
    * @throws {Error}
    */
-  async getListing(listingId, blockInfo) {
+  async getListing(listingId, blockInfo, opts = {}) {
```

Comment:
Probably in a different PR, I would move `blockInfo` into `opts`, because the former is an optional parameter as well.

Reply:
Good thinking. Let me do it in this PR, so we have a minimal as possible `origin-js` API changes.